src/components/app.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-details';
+import SearchForm from './search-form';
+import '../styles/app.scss';
+import axios from 'axios';
+
+const url = 'https://mcr-codes-weather.herokuapp.com/forecast?city=';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: '',
+        country: '',
+      },
+    };
```

Comment:
You can save yourself a line of code by declaring your methods with arrow syntax. `handleForecastSelect = (date) => {}` would bind it's own context so you wouldn't need this line.